src/foam/u2/crunch/CrunchController.js

```diff
@@ -135,11 +135,15 @@ foam.CLASS({
         }).sections);
     },
 
-    function generateAndDisplayWizard(capabilitiesSections) {
+    function generateAndDisplayWizard(capability, capabilitiesSections) {
+      var maybeConfig = capability.wizardConfig
+        ? { config: capability.wizardConfig } : {};
+
       // called in CapabilityRequirementView
       return ctrl.add(this.Popup.create({ closeable: false }).tag({
         class: 'foam.u2.wizard.StepWizardletView',
         data: foam.u2.wizard.StepWizardletController.create({
+          ...maybeConfig,
           wizardlets: capabilitiesSections.wizCaps
```

Comment:
Where in the stepwizardletController is the onclose function being called and passing in the context and ucj?

I was thinking that the config would be created within the stepWizardletController (since it would be referenced on the capability) then from there the context and ucj passed along. 

Is this PR completed or am I missing something?

Reply:
Not completed. Adding `[DNM]` because I'll need #3753 to be merged first to do this properly (otherwise passing the ucj would be non-trivial). As for why this is in CrunchController I can explain on Zoom tomorrow - the short version is StepWizardController is independant of CRUNCH